keyboards/cannonkeys/nearfield/readme.md

```diff
@@ -0,0 +1,23 @@
+# Nearfield
+
+![Nearfield](https://i.imgur.com/hNj44FY.jpg)
+
+The LZ ErGO is a classic ergonomic design, predating the alice layout, and drawing inspiration from the "Microsoft natural ergonomic" series.
+Nearfield expands on this by increasing the rotation of the two halves of the alpha cluster, each half is rotated 15 degrees towards the center of the board.
+This increased angle forces the user to widen their elbow stance, and align their hands straighter with their forearms while typing.
+
+Aesthetic design: Aside from the obvious layout homage to the ergo, retaining the nav cluster,
+the case design draws inspiration from the M0110 keyboard series from Apple.
+
+bog standard mx and alps support pcb, uses unified c3 daughterboard. 100mm JST cable recommended.
+
+ 
+* Keyboard Maintainer: [TJ Campie](https://github.com/tominabox1)
+* Hardware Supported: Nearfield
+* Hardware Availability: Cannonkeys
+
+Make example for this keyboard (after setting up your build environment):
+
+    make nearfield:default
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Comment:
Can you provide examples of how to get the board into bootloader, please?

Reply:
Could you add instructions on how to enter the bootloader, to flash new firmware?  This is needed, as a new user may not be aware of how to enter to bootloader to flash new firmware.

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```